content/en/mTLS (Mutual Transport Layer Security).md

```diff
@@ -0,0 +1,14 @@
+---
+title: mTLS (Mutual Transport Layer Security)
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Mutual TLS (mTLS) is a technique used to authenticate and encrypt messages sent between two [services](https://glossary.cncf.io/service/). Mutual TLS is the standard Transport Layer Security (TLS) protocol but, instead of validating the identity of just one connection, both sides are validated.
+
+## Problem it addresses
+[Microservices](https://glossary.cncf.io/microservices/) communicate over a network and, just like your wifi network, communication in transit over that network can be hacked. mTLS ensures that no unauthorized party can send potentially malicious [API](https://glossary.cncf.io/application_programming_interface/) requests.
+
+## How it helps
+mTLS ensures that traffic is secure and trusted in both directions between a client and server, providing an additional layer of security for users who log in to a network or applications. It also verifies connections with client devices that do not follow a login process, such as Internet of Things (IoT) devices. Attacks like on-path attacks, spoofing attacks, credential stuffing, brute force attacks, malicious API requests, etc. can be prevented by mTLS.
```

Comment:
Let's drop "malicious API requests" as mTLS will only handle ensuring everyone is who they say they are, it won't help if one party is malicious.